src/System.Threading.Timer/tests/TimerChangeTests.cs

```diff
@@ -91,7 +91,7 @@ public void Timer_Change_BeforeDueTime_ChangesWhenTimerWillFire()
         {
             Assert.False(are.WaitOne(TimeSpan.FromMilliseconds(100)), "The reset event should not have been set yet");
             t.Change(TimeSpan.FromMilliseconds(100), TimeSpan.FromMilliseconds(-1));
-            Assert.True(are.WaitOne(TimeSpan.FromMilliseconds(500)), "Should have received a timer event after this new duration");
+            Assert.True(are.WaitOne(TimeSpan.FromMilliseconds(750)), "Should have received a timer event after this new duration");
```

Comment:
Since we expect the event to fire and will fail the test if we don't, should we use a much larger value? e.g. in TimerFiringTests we use MaxPositiveTimeoutInMs set to 30 seconds I believe for cases like this.